### ISSUE TITLE
CoNLL readers to work on InputStreams

### ISSUE THREAD

**Issue by riedelcastro** [wolfe-pack/wolfe#153]
Some of our datasets are saved on the classpath (and packaged in the wolfe-examples.jar). This is useful because users can just add an sbt dependency and get the data, w/o having to setup a data directory etc. It seems that the CoNLL readers require a filename though. Could this be generalized to InputStreams instead?

**Comment by narad** [comment]
Sure, but thinking on the grand scheme of things, would it be better to
have reading from different sources be different static calls?  So you have
a CoNLLReader object with say...readFromFile and readFromStream methods?

**Comment by riedelcastro** [comment]
Yes, I think static convenience methods make sense. I'd imagine that the core method works with streams only, and that the convenience method convert files etc. to streams.
